src/routes/alerts/alerts.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller, Post, HttpCode, Body } from '@nestjs/common';
+import { ApiExcludeController } from '@nestjs/swagger';
+import { Alert } from '@/routes/alerts/entities/alerts.entity';
+import { AlertValidationPipe } from '@/routes/alerts/pipes/alert-validation.pipe';
+import { AlertsService } from '@/routes/alerts/alerts.service';
+
+@Controller({
+  path: '',
+  version: '1',
+})
+@ApiExcludeController()
+export class AlertsController {
+  constructor(private readonly alertsService: AlertsService) {}
+
+  @Post('/alerts')
+  @HttpCode(200)
+  postAlert(
+    @Body(AlertValidationPipe)
+    alertPayload: Alert,
```

Comment:
This should be ok given the current stage of this feature. But the route entities will need to be converted to classes with OpenAPI annotations.

See `src/routes/transactions/entities/add-confirmation.dto.ts` as an example.

Reply:
I've converted these to classes in ffbf4a04e7995592f9eeb14a12cfb84342b54672 but not added any `ApiProperty`s as this will not be exposed to the user.